src/components/HandleRedirectTo/HandleRedirectTo.tsx

```diff
@@ -9,11 +9,10 @@ import * as QueryString from 'query-string';
  */
 export default function HandleRedirectTo() {
   const location = useLocation();
-  const history = useHistory();
   const params = QueryString.parse(location.search);
   const redirectTo = params['redirectTo'] as string;
   if (redirectTo) {
-    history.replace(redirectTo);
+    return <Redirect to={redirectTo} />;
```

Comment:
Using the `Redirect` component makes sure it plays well with the `Redirect` in App.js (overrides it).